subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/configurations/DefaultConfigurationContainer.java

```diff
@@ -64,19 +64,19 @@ public class DefaultConfigurationContainer extends AbstractValidatingNamedDomain
     private final DefaultConfigurationFactory defaultConfigurationFactory;
 
     public DefaultConfigurationContainer(
-        Instantiator instantiator,
-        DependencySubstitutionRules globalDependencySubstitutionRules,
-        VcsMappingsStore vcsMappingsStore,
-        ComponentIdentifierFactory componentIdentifierFactory,
-        ImmutableAttributesFactory attributesFactory,
-        ImmutableModuleIdentifierFactory moduleIdentifierFactory,
-        ComponentSelectorConverter componentSelectorConverter,
-        DependencyLockingProvider dependencyLockingProvider,
-        CollectionCallbackActionDecorator callbackDecorator,
-        NotationParser<Object, ComponentSelector> moduleSelectorNotationParser,
-        ObjectFactory objectFactory,
-        DefaultRootComponentMetadataBuilder.Factory rootComponentMetadataBuilderFactory,
-        DefaultConfigurationFactory defaultConfigurationFactory
+            Instantiator instantiator,
```

Comment:
I am generally not a fan of the double indent here for method parameters (or anywhere tbh). Do you think we can avoid these?

Reply:
IDEA did this, I'll revert it.